test/Microsoft.ML.Tests/OnnxTests.cs

```diff
@@ -59,7 +59,7 @@ public void InitializerCreationTest()
             using (var env = new ConsoleEnvironment())
             {
                 // Create the actual implementation
-                var ctxImpl = new OnnxContextImpl(env, "model", "ML.NET", "0", 0, "com.test");
+                var ctxImpl = new OnnxContextImpl(env, "model", "ML.NET", "0", 0, "com.test", Runtime.Model.Onnx.OnnxVersion.Stable);
```

Comment:
>Runtime.Model.Onnx [](start = 94, length = 18)

it's weird what you can just type OnnxContextImpl but for 
version you need to provide whole namespace, aren't they belong to same namespace?

Reply:
ONNXVersion appears at two different places, legacy C# API and the main ONNX assembly. Unfortunately, both of them are included in this test.

---
In reply to: [223532859](https://github.com/dotnet/machinelearning/pull/947#discussion_r223532859) [](ancestors = 223532859)